skimage/filters/thresholding.py

```diff
@@ -459,14 +459,67 @@ def threshold_isodata(image, nbins=256, return_all=False):
         return thresholds[0]
 
 
-def threshold_li(image):
-    """Return threshold value based on adaptation of Li's Minimum Cross Entropy method.
+def _cross_entropy(image, threshold, bins=np.arange(-0.5, 255.51, 1)):
```

Comment:
Is the default value intended?

Reply:
Yes. Unfortunately you get some nasty aliasing artifacts if you just write 255 or 256...

Reply:
I would prefer to apply your alternative solution, ie, specifying this range before the function call instead as a default value.